src/main/java/seedu/club/logic/commands/SignUpCommand.java

```diff
@@ -0,0 +1,65 @@
+package seedu.club.logic.commands;
+
+import static seedu.club.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_GROUP;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_MATRIC_NUMBER;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_PASSWORD;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_USERNAME;
+
+import seedu.club.logic.CommandHistory;
+import seedu.club.logic.UndoRedoStack;
+import seedu.club.model.Model;
+import seedu.club.model.member.Member;
+
+//@@author Song Weiyang
+/**
+ * Signs up a member when the clubbook is empty
+ */
+public class SignUpCommand extends Command {
+    public static final String COMMAND_WORD = "signup";
+
+    public static final String COMMAND_FORMAT = "login n/ p/ e/ m/ [pic/ ] ";
+
+    public static final String MESSAGE_SUCCESS = "Sign up successful! Please log in again";
+    public static final String MESSAGE_FAILURE = "There are already members in the clubbook. Log In to continue";
```

Comment:
Club Connect already has members of the club. Log in to start using Club Connect.